src/common/stattextcmn.cpp

```diff
@@ -213,8 +213,46 @@ class wxLabelWrapper : public wxTextWrapper
 
 void wxStaticTextBase::Wrap(int width)
 {
+    if (width == m_currentWrap) return;
+    m_currentWrap = width;
+
+    // Allow for repeated calls to Wrap with different values
+    if (m_unwrappedLabel.IsNull()) 
```

Comment:
Please use standard functions instead of wx 1.x ones

```suggestion
    if (m_unwrappedLabel.empty()) 
```